test/unit/metadata.js

```diff
@@ -772,4 +774,67 @@ describe('metadata', function () {
       should(upToDate(doc).errors).be.undefined()
     })
   })
+
+  describe('generated a doc with conflict', () => {
+    it('should get the correct path', () => {
+      const doc = {
+        path: 'docname'
```

Comment:
Why not use `builders.whatever().path('docname').build()`?

Reply:
To do what exactly?

Reply:
Build a valid Metadata object, so we could possibly enable flow?
(i.e. adding `/* @flow */`at the top of this file)

Reply:
the behavior of the function do not need a `Metadata` object but a subset of this kind of object with only a `path`, why do we want to restrict our tests to an implementation instead of a interface?

Reply:
this function takes something with a `path` property and returns something with a `path` property that contains a suffix to stamp the path with a conflict suffix. In fact, this function should take only a file name, as a string.

Reply:
:+1: Agreed!